group_access_tokens.go

```diff
@@ -61,6 +61,31 @@ type ListGroupAccessTokensOptions ListOptions
 // group.
 //
 // GitLab API docs:
+// https://docs.gitlab.com/ee/api/group_access_tokens.html#get-a-group-access-token
+func (s *GroupAccessTokensService) GetGroupAccessToken(gid interface{}, id int, options ...RequestOptionFunc) (*GroupAccessToken, *Response, error) {
+	groups, err := parseID(gid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("groups/%s/access_tokens/%d", PathEscape(groups), id)
+
+	req, err := s.client.NewRequest(http.MethodGet, u, nil, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	gat := new(GroupAccessToken)
+	resp, err := s.client.Do(req, &gat)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return gat, resp, err
+}
+
+// GetGroupAccessToken gets a single Group Access Tokens in a group.
```

Comment:
I think the docs got switched around. This should be `ListGroupAccessTokens`.

Reply:
good catch, thanks. Still puzzled how that could have happened 🤔